ocs_ci/ocs/uninstall.py

```
@@ -185,18 +185,35 @@ def uninstall_ocs():
+            'uninstall.ocs.openshift.io~1confirm-deletion", "value": "true"}]'
+        )
+        result = storage_cluster.patch(
+            resource_name=constants.DEFAULT_CLUSTERNAME,
+            params=confirm_annotation,
+            format_type="json",
+        )
+        if not result:
+            raise CommandFailed(
+                f"StorageCluster '{constants.DEFAULT_CLUSTERNAME}' "
+                "confirm-deletion annotation failed; aborting uninstall"
+            )
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)